src/wymeditor/core.js

```diff
@@ -527,6 +584,239 @@ jQuery.extend(WYMeditor, {
     EXTERNAL_MODULES : {}
 });
 
+WYMeditor.DEFAULT_OPTIONS = {
```

Comment:
I moved this to an externally-referenceable constant so that it can be used in tests.